health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/CreateChecklist.js

```
@@ -0,0 +1,647 @@
+    });
+
+    return result;
+  }
+  let uniqueLocal;
+
+  const payloadData = (data) => {
+
+    data.forEach((question) => {
+      if (question.type.code === "Short Answer") {
+        question.type.code = "String";
+        delete question.options;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)